docs/getting-started.adoc

```diff
@@ -0,0 +1,196 @@
+= Getting Started
+
+This document walks you through the development and deployment of a Node.js application to an OpenShift cluster using `odo`. It guides you on how to push new changes to your application and on other useful features of `odo`.
+
+== Developing and deploying an application to an OpenShift cluster
+
+. link:/docs/installation.md[Install the `odo` binary].
+. Log into an OpenShift cluster:
+
+* If you lack a local development cluster, we recommend using
+https://docs.openshift.org/latest/minishift/getting-started/installing.html[Minishift]
+to deploy a development OpenShift cluster.
++
+.. Run Minishift:
++
+[source,sh]
+----
+$ minishift start
+----
+.. Log into the OpenShift cluster:
++
```

Comment:
there a lot of random `+` throughout the document... please have a look at the file when it's rendered

Reply:
Most of the pluses are required to align the code snippets and nested lists appropriately. I checked the rendered version and took care of the couple of strays, thank you.